joomla/joomla-cms
# Revision to Language String - Article 3 Intro Text

Revise the language string PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_3_INTROTEXT to reflect the number of links set in the menu item Blog.  In the menu item Blog, the option # Links is set to 2.  Where as in the article, it is mentioned as 4.

Pull Request for Issue # .

**### Summary of Changes**
In the Sample Data's Article 3, the number of links set is shown as 4 where as in the menu item Blog it is set as 2.


**### Testing Instructions**

(a) Create a new installation of Joomla!

(b) Log into the Back-end

(c) In the Control Panel click on the link "Blog Sample data" (that is in the module titled "SAMPLE DATA"

(d) After the installation of the Sample Data is successful, open the menu item "Blog" (that is in the Menu titled "Main Menu Blog") for Edit and go to the tab "Blog Layout". Notice that the value shown for the option "# Links" is set to 2 (Two).

![sample_data_blog_menu_item_no_of_links_as_2](https://user-images.githubusercontent.com/4947580/30251689-a0dee480-9682-11e7-876a-560c08610678.jpg)

(e) See the new site created in a separate browser tab

(f) Click on the Link "Blog" and scroll to see the screen to see the contents of the blog.  Notice that in the article titled "About your home page", the number of links is mentioned as 4.

![sample_data_blog_page_showing_no_of_links_as_4](https://user-images.githubusercontent.com/4947580/30251706-115699ce-9683-11e7-84fc-8aa00cc314ef.jpg)

**### Expected result**

In the article titled "About your home page" the number of link must specify the value 2 - NOT 4.

![sample_data_blog_page_showing_no_of_links_as_2](https://user-images.githubusercontent.com/4947580/30251755-cdbc1788-9683-11e7-9178-a87e92671276.jpg)

**### Actual result**

See the step (f) in the section ### Testing Instructions


### Documentation Changes Required
